.mvn/extensions.xml

```diff
@@ -2,6 +2,6 @@
   <extension>
     <groupId>org.eclipse.tycho.extras</groupId>
     <artifactId>tycho-pomless</artifactId>
-    <version>2.5.0</version>
+    <version>3.0.4</version>
```

Comment:
```suggestion
    <version>4.0.4</version>
```
The latest Tycho version as of today is [4.0.4](https://github.com/eclipse-tycho/tycho/releases/tag/tycho-4.0.4)